src/components/ebay-menu-button/index.marko

```diff
@@ -90,8 +90,8 @@ $ var isOverflowVariant = state.variant === "overflow";
         onMenu-change("handleMenuChange")
         onMenu-select("handleMenuSelect")
         key="content">
-        <for|item| of=state.items>
-            <ebay-menu-item(item)/>
+        <for|item,index| of=input.items>
+            <ebay-menu-item  ...item checked=state.checkedItems[index] />
```

Comment:
Internally we should change this to `<@item>` to avoid work when we deprecate the nested tag transform.